class.jetpack-jitm.php

```diff
@@ -132,18 +163,46 @@ function photon_msg() {
 	 * @since 3.8.2
 	 */
 	function manage_pi_msg() {
-		if ( current_user_can( 'jetpack_manage_modules' ) ) {
-			$normalized_site_url = Jetpack::build_raw_urls( get_home_url() );
-			$manage_active = Jetpack::is_module_active( 'manage' );
-			$manage_pi_dismissed = isset( self::$jetpack_hide_jitm['manage-pi'] );
+		$normalized_site_url = Jetpack::build_raw_urls( get_home_url() );
+		$manage_active       = Jetpack::is_module_active( 'manage' );
+		// If it's not an array, it means no JITM was dismissed
+		$manage_pi_dismissed = self::is_jitm_dismissed();
```

Comment:
This is no longer necessary with the early check added in 8ae5df6e6d94b9139f2ed8861cf34af9c589083e